### DISCUSSION TITLE
Different entry & exist VPNs

### DISCUSSION THREAD

**Discussion by olddustysocksunderthecouch** [OutlineFoundation/outline-sdk#76] [Q&A]
I'm currently visiting a country where VPNs are mostly banned. Unfortunately, the VPN my employer uses is one of the banned ones.

I'd like to set up a VPN on an EC2 instance, and then it forwards to my employer's VPN. Can this be done with Outline? 

I'd welcome any alternative ideas! I'm stuck and might have to move countries which I really don't want to have to do....

**Comment by daniellacosse** [comment]
Here's a tutorial! https://github.com/net4people/bbs/issues/126#issuecomment-1257116005
